sklearn/linear_model/logistic.py

```diff
@@ -1789,3 +1789,35 @@ def fit(self, X, y, sample_weight=None):
 
         self.C_ = np.asarray(self.C_)
         return self
+
+    def score(self, X, y, sample_weight=None):
+        """Returns the score using the `scoring` option on the given
+        test data and labels.
+
+        Parameters
+        ----------
+        X : array-like, shape = (n_samples, n_features)
+            Test samples.
+
+        y : array-like, shape = (n_samples) or (n_samples, n_outputs)
+            True labels for X.
+
+        sample_weight : array-like, shape = [n_samples], optional
+            Sample weights.
+
+        Returns
+        -------
+        score : float
+            Score of self.predict(X) wrt. y.
+
+       """
+
+        scoring = self.scoring
+        if isinstance(scoring, six.string_types):
+            scoring = get_scorer(scoring)
+
+        if scoring is None:
+            return (super(LogisticRegressionCV, self)
+                    .score(X, y, sample_weight=sample_weight))
+
```

Comment:
we should probably raise a ChangedBehaviorWarning here.

Reply:
I think that scoring should default to "accuracy" in the init so you don't need this branching.

Reply:
I think that the scoring param in the init should default to "accuracy" so you don't need this branching.

Reply:
The other sklearn classes that accepts a `scoring` parameter, stores it into `self.scoring` without considering a default. 

To avoid branching, I added the following to the `score` function:

```python
scoring = self.scoring or 'accuracy'
```

This hardcodes accuracy as the default scorer.